dev-python/gitdb2/gitdb2-2.0.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="GitDB is a pure-Python git object database"
+HOMEPAGE="
+	https://github.com/gitpython-developers/gitdb
+	https://pypi.python.org/pypi/gitdb2"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	!dev-python/gitdb[${PYTHON_USEDEP}]
+	>=dev-python/smmap2-2.0.1[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
+# Testsuite require files from a git repo
```

Comment:
Figured it out. Do the following:

1. Create a new git repo.
2. Fill it with a lot of blobs (2000+) — you can just use `{1..2000}` to create 2000 different files, add them and commit ;-).
3. Convert the repo to bare.
4. Set `GITDB_TEST_GIT_REPO_BASE` to that repo.
5. Set `TRAVIS` to `1` to disable performance tests (they rely on using exact gitdb repo and they only print performance).